static/app/components/button.tsx

```diff
@@ -167,13 +167,13 @@ const getFontWeight = ({priority, borderless}: StyledButtonProps) =>
   `font-weight: ${priority === 'link' || borderless ? 'inherit' : 600};`;
 
 const getBoxShadow =
-  (active: boolean) =>
+  (theme: Theme, active: boolean) =>
   ({priority, borderless, disabled}: StyledButtonProps) => {
     if (disabled || borderless || priority === 'link') {
       return 'box-shadow: none';
     }
 
-    return `box-shadow: ${active ? 'inset' : ''} 0 2px rgba(0, 0, 0, 0.05)`;
+    return `box-shadow: ${active ? 'inset' : ''} ${theme.dropShadowLight}`;
```

Comment:
Do we also need to update the LESS based styles for buttons?

Reply:
Think that would fit better in a separate future PR for refactoring LESS styles for buttons, since it's not just the drop shadows that are out of date 😅